app/src/main/java/org/covidwatch/android/data/NtpTime.kt

```diff
@@ -0,0 +1,19 @@
+package org.covidwatch.android.data
+
+import android.content.Context
+import com.lyft.kronos.AndroidClockFactory
+import com.lyft.kronos.KronosClock
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneId
+
+class NtpTime (context : Context) {
+    private val kronosClock : KronosClock = AndroidClockFactory.createKronosClock(context)
+
+    private fun currentTimeMillis() : Long = kronosClock.getCurrentTimeMs()
+
+    fun syncInBackground() = kronosClock.syncInBackground()
+
+    fun nowAsLocalDate() : LocalDate =
+        Instant.ofEpochMilli(currentTimeMillis()).atZone(ZoneId.systemDefault()).toLocalDate()
```

Comment:
`ZoneId.systemDefault()` introduces the same problem that we tried to eliminate -- it tights the time of the app to the users' settings. I was able to reproduce a one day shift with a timezone that have the next day relatively to mine. It's not that big as before but still a problem.

One way to resolve is just to remove `atZone(ZoneId.systemDefault())`.

Reply:
removed notion of LocalDate from NtpTime to avoid confusion / simplify the code